daemon/call.c

```diff
@@ -2466,8 +2466,10 @@ int monologue_offer_answer(struct call_monologue *other_ml, GQueue *streams,
 	struct call_monologue *monologue = other_ml->active_dialogue;
 	struct endpoint_map *em;
 	struct call *call;
+	unsigned int bridge_count;
```

Comment:
You can eliminate this, as g_queue_pop_head returns NULL if the queue is empty anyway, which translates to zero